Dockerfile

```diff
@@ -6,31 +6,35 @@ RUN  export DEBIAN_FRONTEND=noninteractive
 ENV  DEBIAN_FRONTEND noninteractive
 RUN  dpkg-divert --local --rename --add /sbin/initctl
 
-RUN apt-get -y update
-RUN apt-get -y install gnupg2 wget ca-certificates rpl pwgen
+RUN apt-get -y update; apt-get -y install gnupg2 wget ca-certificates rpl pwgen
```

Comment:
Update and install needs to be on the same line. Otherwise it will be cached and the actual package location might change in the future.